scapy/contrib/lldp.uts

```diff
@@ -0,0 +1,168 @@
+from enum import test
+% LLDP test campaign
+
+#
+# execute test:
+# > test/run_tests -P "load_contrib('lldp')" -t scapy/contrib/lldp.uts
+#
+
++ Syntax check
+= Import the LLDP layer
+from scapy.contrib.lldp import *
```

Comment:
This is not needed. Unit tests should be launched as:
`./test/run_tests -P "load_contrib('gtp')" ../scapy/contrib/lldp.uts`